coverage/src/BamView-WebComponent.js

```diff
@@ -100,6 +100,32 @@ button:hover {
     position: relative;
 }
 
+/* Styles for the reset button */
+.reset-button:hover circle {
```

Comment:
You should be able to combine `reset-button` and `chromosome` into a single class. Maybe `region-change-button`?

Reply:
The reset circle button and chromosome button are independent of each other. They serve different purposes. In my zoomToChromosome function, I will remove the chromosome-button-small, and redraw the chromosome-button-big. The reset button will not change. So we cannot use the same class name for them here. 
<img width="685" alt="Screenshot 2024-06-26 at 4 23 05 PM" src="https://github.com/iobio/iobio-charts/assets/60582494/951649d3-042f-47a7-968d-7e65922f9349">

Reply:
That's fine if you need different classes for them. But those specific styles are duplicated. In this case they have the same meaning, which is that a button should not have selectable text. You should have a single class for that behavior. You can apply more than one class to an element. So you can keep the separate classes, but move the duplicated styles into a single class, and also add that class to the elements.

The more I think about this, you should probably be using `<button></button>` elements for this, since that's what they are. You just need to style them to match the visual representation you want.

Reply:
That makes sense. Thanks for your suggestion. I deleted the duplicated CSS and added the new class to group the style.

Reply:
Excellent, thanks. Also I just realized my earlier suggestion to use HTML `<button>` elements makes no sense because this is all in SVG.